clarified dependency installation process (#3)
boom100100/ats-beater: README.md

## Description
Appends the job description of the current webpage to your resume.

## Dependencies

> The native host for this extension is the [Local and Website PDF Merger](https://github.com/boom100100/local_and_website_pdf_merger). Follow instructions in the [readme](https://github.com/boom100100/local_and_website_pdf_merger/blob/main/README.md) to complete setting it up.

Author: ```suggestion
The native host for this extension is the [Local and Website PDF Merger](https://github.com/boom100100/local_and_website_pdf_merger). Follow instructions in the [readme](https://github.com/boom100100/local_and_website_pdf_merger/blob/main/README.md) there to install the official version.
```
